Makefile

```diff
@@ -159,6 +159,12 @@ redeploy: load-images # Redploy apex after images changes
 	@kubectl rollout restart deploy/apiserver -n apex
 	@kubectl rollout restart deploy/frontend -n apex
 
+.PHONY: redeploy-db
+redeploy-db: redeploy-db # Delete and bring up a new apex database
```

Comment:
I know the docs include "Delete" here, but the `redeploy` target isn't destructive to data, so someone might mistakenly think `redeploy-db` restarts the DB, but with the same existing data. Maybe we could call it something like `recreate-db` ?